src/Aspire.Hosting.Kafka/KafkaBuilderExtensions.cs

```diff
@@ -25,11 +26,34 @@ public static IResourceBuilder<KafkaServerResource> AddKafka(this IDistributedAp
         var kafka = new KafkaServerResource(name);
         return builder.AddResource(kafka)
             .WithEndpoint(targetPort: KafkaBrokerPort, port: port, name: KafkaServerResource.PrimaryEndpointName)
+            .WithEndpoint(targetPort: 9093, name: KafkaServerResource.InternalEndpointName)
```

Comment:
Work having a constant for this too? Makes it more self documenting?

Reply:
added a constant an used it.